custom_components/tahoma/lock.py

```diff
@@ -46,29 +48,21 @@ def update(self):
             return
 
         self.controller.get_states([self.tahoma_device])
-        self._battery_level = self.tahoma_device.active_states["core:BatteryState"]
-        self._name = self.tahoma_device.active_states["core:NameState"]
         if (
-            self.tahoma_device.active_states.get("core:LockedUnlockedState")
+            self.tahoma_device.active_states.get(CORE_LOCKED_UNLOCKED_STATE)
             == STATE_LOCKED
```

Comment:
`STATE_LOCKED` is a constant from Home Assistant, not from the TaHoma component. Do we want to use those? My preference would be to have a specific one from the component, to keep it away from HA core.

Reply:
done